### DISCUSSION TITLE
Why no dev activity/releases anymore?

### DISCUSSION THREAD

**Discussion by wottpal** [alephjs/aleph.js#559] [General]
Hey there, I have just dived into `deno` and found this _super exciting_ framework. But I'd like to know whether it's a good bet to build something with it, as the development activity has slowed down. Are there any particular reasons?

**Comment by ije** [comment]
I am trying to come back to this project as soon as possible, currently i am crazy busy on other things...

**Comment by AddictArts** [comment]
Just came to this project after discovering it. Thanks for any future insight and possibly reaching 1.0.0. Understood that help may be needed.

**Comment by mattvr** [comment]
Better Q is: how can we help? This is a lot for one dev.

**Comment by AddictArts** [comment]
@mattvr directed to me or all who come here? If me I completely agree that help may be needed. @ije had replied, it has been months, most likely crazy busy is the status and that is understood. I am just a new fan of the work. I've edited my reply above as to not imply I think anyone owes me an update or more work.

**Comment by ije** [comment]
@AddictArts thanks for interesting, unfortunately i still can not manage time to the project update, did you try fresh or ultrajs? them are great projects!

- https://fresh.deno.dev/
- https://ultrajs.dev/

**Reply by AddictArts** [reply]
Thanks so much and for the tips. I am checking out ultrajs. Discussion below is just for anyone else curious. Thanks again @ije 

I mentioned react over on fresh and the backlash was disappointing; however, this was not the maintainers, just the audience and strange reaction. Fresh is doing good things, for me though the `preact` and `preact/compat` is a tough sell for existing work already in react.
